go-controller/pkg/ovn/address_set/address_set.go

```diff
@@ -42,10 +42,8 @@ type AddressSetFactory interface {
 
 // AddressSet is an interface for address set objects
 type AddressSet interface {
-	// GetIPv4HashName returns the hashed name for v4 address set
-	GetIPv4HashName() string
-	// GetIPv6HashName returns the hashed name for v4 address set
-	GetIPv6HashName() string
+	// GetASHashName returns the hashed name for ipv6 and ipv4 addressSets
+	GetASHashName() (string, string)
```

Comment:
@JacobTanenbaum since it's returning tow strings now, can we do GetASHashNames()? (eg plural)

Reply:
done